pkg/scheduler/algorithmprovider/registry.go

```diff
@@ -151,18 +124,10 @@ func getDefaultConfig() *schedulerapi.Plugins {
 	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeCapacityPriority) {
```

Comment:
can you please move this to `applyFeatureGates` func below please.

Reply:
Of course, done